### DISCUSSION TITLE
Inquiring and Questions

### DISCUSSION THREAD

**Discussion by Mublin** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#8] [Outreachy 2022] [answered]
Assalamu Alaikum @BatoolMM  @alswajiab and the rest of the community, I am Mubarak Muhammad, from Nigeria and an outreachy intern interested in working and improving your work, I have tried to fill out the form but it is not working. looking forward to working with you. Have a nice day

**Comment by BatoolMM** [comment, marked as answer]
Hi @Mublin, 
Apologies for the broken link, is [this one](https://form.jotform.com/221813832742353) working for you?

**Comment by Mublin** [comment]
Thank you for your email, yes, it is working and I have completed the form
looking forward to working with you and the team.

**Comment by BatoolMM** [comment]
Looking forward to seeing your contribution!

**Reply by Mublin** [reply]
will I have to record my contribution on the pc screen

**Comment by Mublin** [comment]
Thank you! Where do I start from here?
